### PR TITLE
Mocks

### PR DESCRIPTION
**Pull request by kathytbui**
Created several mocks to decrease test time

### CODE REVIEW

File: test/stat_tracker_test.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 require "./lib/stat_tracker"
 require "minitest/autorun"
 require "minitest/pride"
+require "mocha/minitest"
```
Comment (kathytbui) [inline test/stat_tracker_test.rb:4]: Added mocks

File: test/stat_tracker_test.rb [hunk 2]
```diff
@@ -104,6 +105,9 @@ def test_it_can_get_all_team_scores
     assert_equal true, @@stat_tracker.team_scores.all? do |team_id, scores|
       team_id.is_a?(String) && scores.is_a?(Array)
     end
+    @@stat_tracker.stubs(:scores).returns({"3"=>[2,4]})
```
Comment (kathytbui) [inline test/stat_tracker_test.rb:108]: stubbed out helper method

File: test/stat_tracker_test.rb [hunk 3]
```diff
@@ -450,15 +367,19 @@ def test_it_gets_worst_coach
...
-
+    @@stat_tracker.expects(:game_ids_in_season).returns([2014030411, 2014030412])
```
Comment (kathytbui) [inline test/stat_tracker_test.rb:366]: stubbed out return so that we didn't have to deal with that huge array

File: test/stat_tracker_test.rb [hunk 4]
```diff
@@ -450,15 +367,19 @@ def test_it_gets_worst_coach
...
-    assert_equal expected, @@stat_tracker.game_ids_in_season("20142015")
+    assert_equal 1319, @@stat_tracker.game_ids_in_season("20142015").count
```
Comment (kathytbui) [inline test/stat_tracker_test.rb:376]: stubbed out helper method return to use smaller testable array.

**Review by Jonathan-M-Wilson — APPROVED** [review]
These mocks and stubs look super good! Your notes were awesome and everything here looks good to go!

File: test/stat_tracker_test.rb [hunk 5]
```diff
@@ -445,15 +363,19 @@ def test_it_gets_worst_coach
...
-
+    @@stat_tracker.expects(:game_ids_in_season).returns([2014030411, 2014030412])
```
Comment (ajtran303) [inline test/stat_tracker_test.rb:366]: I can see why you did this

**Pull request merged**